third_party/move/move-vm/runtime/src/storage/ty_depth_checker.rs

```diff
@@ -138,7 +138,8 @@ where
             | Type::U128
             | Type::U256
             | Type::Address
-            | Type::Signer => check_depth!(0),
+            | Type::Signer
+            | Type::Function { .. } => check_depth!(0),
```

Comment:
Maybe add a brief comment for this arm?

Reply:
Added, same for one more arm below where a constant is returned.